emojiList.go

```diff
@@ -0,0 +1,3 @@
+package main
```

Comment:
I don't think this file should be here, since mage will build it for us

Reply:
this was removed, i was mostly keeping a copy of it in some revision for reference, in case i needed to search it.  It won't be in the PR